docs/whatsnew/0.8.rst

```diff
@@ -0,0 +1,140 @@
+.. doctest-skip-all
+
+.. _whatsnew-0.8:
+
+**************************
+What's New in SunPy 0.8?
+**************************
```

Comment:
ah, I've found the source: https://docs.python.org/devguide/documenting.html#sections

(the "normal" rst docs had recommended a different sequence).